include/modem/lte_lc.h

```diff
@@ -464,6 +464,16 @@ enum lte_lc_modem_evt {
 	LTE_LC_MODEM_EVT_OVERHEATED,
 };
 
+/** @brief type of factory reset to perform */
+enum lte_lc_factoryreset_type {
+	/** Reset all modem data to factory settings */
+	LTE_LC_FACTORYRESET_ALL = 0,
+	/** Reset user-configurable data to factory settings */
+	LTE_LC_FACTORYRESET_USER = 1,
+	/** Invalid value */
+	LTE_LC_FACTORYRESET_INVALID = 99,
```

Comment:
Not sure what the purpose of this type is.

Reply:
I used it in the modem shell as a placeholder value - I presume its best to move its definition somewhere there